README.md

```diff
@@ -1 +1,78 @@
-# Project-for-code-review
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+# 📖 [Project-for-code-review] <a name="about-project"></a>
+
+**[Project-for-code-review]** is a project built as a student of Microverse during the admission into the programe
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+This project was made with:
+
+- html
+- css
+
+### Key Features <a name="key-features"></a>
+
```

Comment:
![image](https://user-images.githubusercontent.com/85732251/206142829-93ee2e9d-5d93-46fc-b3a2-544f7f5b4272.png)

## Highlights
✔️ No Linter Errors 👍
✔️ Professional PR 👍
✔️ Awesome design 👍

## Required Changes ♻️
Check the comments under the review.

Reply:
**Great work done so far!** 👍 

I think it will be professional if you make your **Key features** distinct.